docs/DeveloperGuide.adoc

```diff
@@ -358,25 +699,24 @@ _{More to be added}_
 [[mainstream-os]] Mainstream OS::
```

Comment:
Could add terms like 'repeater' to the glossary as people might be unsure of what it means.